students/mariusz_michalczyk/lesson_01_basics/Digital_clock.py

```diff
@@ -0,0 +1,8 @@
+from math import *
+passed_minutes = int(input("Enter minutes: "))
+day_minutes = passed_minutes % 1440
+print (str(day_minutes // 60) + " " +  str(day_minutes % 60))
+
+
+
```

Comment:
Those empty lines should probably by caught by linter. Your IDE didn't complain?